ui/src/app/features/application-list/application-list/application-list.component.scss

```diff
@@ -31,15 +41,55 @@
   }
 
   .table-header-container {
+    justify-content: space-between;
```

Comment:
please follow CSS order when declaring props. Good example https://9elements.com/css-rule-order/

i.e. first define display: flex; and then properties related to it (justify-content, align-items)